src/main/java/org/apache/sysds/utils/NativeHelper.java

```diff
@@ -146,7 +146,20 @@ private static boolean isSupportedArchitecture() {
 		LOG.info("Unsupported architecture for native BLAS:" + SystemUtils.OS_ARCH);
 		return false;
 	}
-	
+
+	/**
+	 * Note: we only support Windows and Linux at the moment.
+	 *
+	 * @return true if operating system is supported
+	 */
+	private static boolean isSupportedOS() {
+		if(SystemUtils.IS_OS_LINUX || SystemUtils.IS_OS_WINDOWS) {
```

Comment:
could it not also include UNIX?
for mac support?
If not it would make sense to add a task on JIRA fro this

Reply:
We don't have Mac support anymore. We can put it on the wish list in JIRA :wink: